.circleci/config.yml

```diff
@@ -18,39 +18,43 @@ jobs:
       - checkout: *post_checkout
       - restore_cache: *_root_package_lock_key
       - run: yarn install --frozen-lockfile
+      - persist_to_workspace:
```

Comment:
So the idea here is that we persist the post-install filesystem to the workflow-only FS layer, from which all other jobs inherit, then instead of caching `node_modules` proper we cache the yarn cache?

Reply:
I compared two CI runs to see how the install speed changed and it looks like the initial install took a while longer (~2m vs ~1m) but the other jobs didn't really change. 

I'm not too worried about the 1m difference and tbh it might just be because this particular run did not hit the yarn cache.

Reply:
The advantage is that a setup is generated once in the first install job and then the reused pristinely within each of the workflow jobs.  It essentially guarantees each job within a particular workflow execution is run against an identical setup.  There's also the advantage that external network access (npm registry, github, etc.) is limited to the first step (excepting individual test needs within a job).

Reply:
Yeah having the exact same copy is a really good guarantee.